CMS/cms-basics

```diff
@@ -0,0 +1,345 @@
+---
+title: CMS basics
+---
+
+## Introduction
+
+Skygear CMS aims to provide a user friendly-interface for business user to update the content of the app. As a developer, you can configure what to display / be editable in the CMS with the YML file at the Developer Portal.
+
+Some of you may wonder what's the difference between the CMS and the data browser, here's a summary.
+
+
+|Difference	|Database Browser	|CMS	|
+|---	|---	|---	|
+|Target user	|Developers who need to view and alter database data and schema.	|System Admin or managers who need to view and update content of the system.	|
```

Comment:
System Admin => Business users who need to view and update content of the system

Reply:
ok!